charts/ccd-docker-user-profile-importer/values.yaml

```diff
@@ -0,0 +1,17 @@
+image: 'hmctspublic.azurecr.io/ccd/user-profile-importer:latest'
+memoryRequests: '512Mi'
+cpuRequests: '250m'
+memoryLimits: '2048Mi'
+cpuLimits: '1500m'
+userProfileUrl: 
+users:
+jurisdictions:
+waitHosts: ""
+waitHostsTimeout: 300
+microservice: ccd_definition
+verbose: "false"
+userProfileDatabaseHost: ccd-user-profile-api-postgres-db-{{ .Values.global.environment }}.postgres.database.azure.com
```

Comment:
doesnt make sense to point to Azure PAS DB when passwords are pointing to local postgresql

Reply:
modified the value now